wpimath/src/main/java/edu/wpi/first/math/kinematics/ChassisSpeeds.java

```diff
@@ -103,7 +103,9 @@ public Twist2d toTwist2d(double dtSeconds) {
    * @param omegaRadiansPerSecond Angular velocity.
    * @param dtSeconds The duration of the timestep the speeds should be applied for.
    * @return Discretized ChassisSpeeds.
+   * @deprecated Use instance method instead.
    */
+  @Deprecated
```

Comment:
```suggestion
  @Deprecated(forRemoval = true, since = "2025")
```
(Same applies to the other 8 deprecated methods/overloads)

Reply:
Done!